tests/meshcop/commissioner.hpp

```diff
@@ -111,6 +115,14 @@ class Commissioner
      */
     bool IsValid(void) const;
 
+    /**
+     * This method returns whether the commissioner petition succeeded
+     *
+     * @returns inner commisioner state is kStateAccepted
+     *
+     */
+    bool IsCommissionerAccepted();
```

Comment:
All void functions or methods shall explicitly declare and specify the void type keyword.